sdk/ml/azure-ai-ml/azure/ai/ml/operations/_model_operations.py

```diff
@@ -87,7 +87,9 @@ class ModelOperations(_ScopeDependentOperations):
     :type service_client: typing.Union[
         ~azure.ai.ml._restclient.v2023_04_01_preview._azure_machine_learning_workspaces.AzureMachineLearningWorkspaces,
         ~azure.ai.ml._restclient.v2021_10_01_dataplanepreview._azure_machine_learning_workspaces.
-        AzureMachineLearningWorkspaces]
+        AzureMachineLearningWorkspaces
+
+    ]
```

Comment:
Is this whitespace needed? Seems superfluous.

Reply:
Yes, annoyingly Sphinx gets confused with listing things inside brackets/parentheses. The only way I could get it to stop complaining about no line after was to add this spacing. From the docs I generated locally, it shouldn't affect the appearance in our reference docs.

Reply:
Update: I found out another way to fix the error that allowed me to remove the whitespace :) 

I confirmed it looks correct in the doc build.